.eslintrc.json

```
@@ -50,13 +51,15 @@
       },
       "rules": {
         "@typescript-eslint/no-import-type-side-effects": "error",
+        "@typescript-eslint/strict-boolean-expressions": "off",
```

Let's set this and the others to warn for now and fix the issues in another PR.
```suggestion
        "@typescript-eslint/strict-boolean-expressions": "warn",
```